docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
+
+**An example**:
+Imagine that you are interested in knowing whether smoking (our risk factor of interest) influences the risk of coronary heart disease (CHD) in men aged 18 to 64 years. You conduct a cohort study with an exposed group of male smokers in this age group from the general population, and compare their risk of CHD after a number of years to that of an unexposed group of non-smoking males in the same age group. You would expect to find a higher incidence of CHD among smokers. However, males who smoke are also likely to have a higher alcohol intake, which also increases the risk of CHD. Thus, the effect that you observe for the smoking-CHD association is **mixed up** with the effect of the association between alcohol intake and CHD. Without taking account of differences in alcohol intake between smokers and non-smokers, the magnitude of the smoking-CHD association estimated from the study may be higher than the **true** value, leading to incorrect conclusions.
+
+The criteria for alcohol to be a confounding factor for the smoking-CHD relationship is met because:
+
+  1. Alcohol intake is a risk factor for CHD in non-smokers. In other words the relationship between alcohol and CHD is not dependent on smoking status.  
+  2. Smokers generally have a higher alcohol intake (in most populations). 
+  3. Finally, alcohol intake is not on the causal pathway between smoking and CHD (smoking does not in itself cause people to drink more alcohol). 
+
+.. image:: smoking_triangle.svg
+
+Note that the dotted line between smoking and alcohol indicates that we do not expect a 'causal' link between the smoking and alcohol. Rather, it indicates that the two are associated in the population.
+
+**How to identify confounding?**
+
+We look for confounding by stratifying (splitting) the data according to the proposed confounding factor and then examining the measures of effect of the exposure on the outcome in the different strata separately. If the stratum-specific measures of effect are similar to each other, but different from the crude measure of effect, this is evidence for confounding. We can now consider a numerical example to demonstrate how you can look for confounding in data from an epidemiological study.
+
+**An example**:
```

Comment:
I think, because in this case, coffee is not causally related to the outcome, this is a confusing example that we might not want to include.

While this does fit the three points of the definition (1, smoking is a risk factor for cancer, 2. smoking is correlated with coffee consumption, and 3., smoking does not cause coffee consumption), given that coffee isn't a risk factor for cancer, would we ever put it in one of our models? and does this still fall under the purview of confounding?

Reply:
I see what you're saying here, Beatrix. I think examples like these are shown in intro epi lectures to demonstrate that if you don't account for confounding, you can find *spurious* relationships and conclude that coffee causes pancreatic cancer even if that's not the case (as a warning to be carefully when analyzing primary data collected in epi studies!). I agree though that it is a little less relevant for our purposes in that we will pretty much never (shouldn't) be dealing with exposures that aren't causally related to the outcome. I'm fine with keeping the coffee example because it's already written out, but maybe we could just include a note about this somewhere?

Reply:
ya! :) 

In our models, we never are involved with proving causality. but we need to make an assessment of whether the reported effect sizes are real or not. Or whether the reported effect sizes are as big/small as they claim- maybe some of that effect is coming from a confounder that they failed to adjust for (which happens often!) and we are biasing our models results.